src/Settings/SettingsHelper.php

```diff
@@ -354,4 +548,42 @@ public function getIntegrationFieldsValue(array $dbSettingsValue, array $formFie
 
 		return $formFields;
 	}
+
+	/**
+	 * Create our custom tables, used on plugin activation.
+	 *
+	 * @return void
+	 */
+	public function createDbTables(): void
+	{
+		global $wpdb;
+
+		$tableNameSettings = "{$wpdb->prefix}" . Config::getDbSettingsName();
+		$tableNameOptions = "{$wpdb->prefix}" . Config::getDbOptionsName();
+
+		$sqlSettings = "CREATE TABLE {$tableNameSettings} ( 
+			meta_id BIGINT(20) UNSIGNED NOT NULL AUTO_INCREMENT,
+			post_id BIGINT(20) UNSIGNED NOT NULL DEFAULT '0',
+			meta_key VARCHAR(255) NULL DEFAULT NULL,
+			meta_value LONGTEXT NULL DEFAULT NULL,
+			PRIMARY KEY (meta_id),
+			INDEX post_id (post_id),
+			INDEX meta_key (meta_key)
```

Comment:
Will it be necessary to query based on `meta_value` or a `meta_key` and `meta_value` combination? If so, I'd suggest adding an index on `meta_value` in the former case, that is, on `meta_key+meta_value` in the latter case. A good read regarding such indices is the WPVIP docs on the WordPress `postmeta` table: https://docs.wpvip.com/technical-references/code-quality-and-best-practices/querying-on-meta_value/